msteams-platform/concepts/device-capabilities/browser-device-permissions.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Device permissions for the browser
+keywords: teams apps capabilities permissions
+description: Securely bring back device permissions support for apps in our web client
+localization_priority: Normal
+ms.topic: how-to
+---
+
+# Device permissions for the browser
+
+Users can grant consent to each app and consent to device permissions. Currently, when permissions are granted for one app, it's also granted for all other apps. An update made in Chromium browser deprecated HTML5 permissions from cross-origin iframes. Chromium provides a way to allow iframes to request these permissions:
+
+* Camera
+* Microphone
+* Location access
+
+The request or consent dialog is for the ancestor, in this case, [teams.microsoft.com](https://teams.microsoft.com/). For example, if an app requests microphone access and access is granted, all other apps in Teams automatically grants device permissions for microphone. The user will never see any prompt from these apps. The user isn't prompted as the permission is already granted.
```

Comment:
This is unclear. The request should be for the individual app, not the Teams app.